lib/glific/flows.ex

```diff
@@ -211,10 +217,24 @@ defmodule Glific.Flows do
 
       flow = get_status_flow(flow)
 
-      {:ok, flow}
+      if Map.has_key?(attrs, :add_role_ids),
+        do: update_flow_roles(attrs, flow),
+        else: {:ok, flow}
```

Comment:
This does not match the return type of this function. Also why we changed it from Flow.t() to map?

Reply:
reverted the changes